totalspineseg/utils/iterative_label.py

```diff
@@ -34,6 +36,10 @@ def main():
         '--output-segs-dir', '-o', type=Path, required=True,
         help='Folder to save output segmentations.'
     )
+    parser.add_argument(
+        '--locs-dir', '-l', type=Path, default=None,
```

Comment:
maybe add `required=False,` as this is an optional flag

Reply:
Thanks a lot for reviewing the PR and the comment,
Are you sure it needed? Because for what I understand the default is False for non positional arguments.

https://docs.python.org/3/library/argparse.html#required

Reply:
Yes, if you don't specify, the `required` argument is set to false